README.md

```diff
@@ -21,23 +21,25 @@ example demonstrates the usage of HTTPLimiter for rate-limiting access
 to an http.Handler to 20 requests per path per minute with bursts of
 up to 5 additional requests:
 
-	store, err := memstore.New(65536)
-	if err != nil {
-		log.Fatal(err)
-	}
+```go
+store, err := memstore.New(65536)
+if err != nil {
+	log.Fatal(err)
```

Comment:
Not sure what you mean - the extra level of indentation that was there before was to render this as a codeblock, or was there any additional purpose? The block is still indented with tabs & renders the same (apart from the additional syntax highlighting :)